aether-utils/aether-client/setup.py

```diff
@@ -19,15 +19,16 @@
 import os
 from setuptools import setup, find_packages
 
-# allow setup.py to be run from any path
-os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
-
 setup(
     name='aether.client',
-    version='0.0.0',
+    version=os.environ['VERSION'],
```

Comment:
Same comment as in the last attempt https://github.com/eHealthAfrica/aether/pull/360#discussion_r216688138